docs/tutorials/Bonding_and_Charge/mulliken_population.md

```diff
@@ -56,4 +56,130 @@ Si              2     1.356   2.644   0.000   0.000   4.000     0.000
+======================================================================
+```
+The results are rather similar but there are a couple of interesting things to note:
+
+- The ratio of the populations in s and p orbitals is closer to 1:3, which is expected for sp^3^ hybridization - this indicates that the bonds are much more perfectly hybridized, meaning it's more overlapped: this indicates stronger bonding, as well as showing that silicon is a semimetal
+- The population of electrons in the bonds is the same (indicating that the same type of bonding is present), but the bond length is smaller - again indicating more overlap and thus stronger bonding.
+
+Now we will compare it with GaAs. The same procedure is used, except the lattice length is now 3.93 and the atoms in ```positions_frac``` should be ```Ga``` and ```As``` (it doesn't matter which one goes in which position/line).
+
+We get an output looking like this:
+
+```
+Atomic Populations (Mulliken)
+-----------------------------
+Species          Ion     s       p       d       f      Total   Charge (e)
+==========================================================================
+Ga              1     1.180   1.743   9.994   0.000  12.917     0.083
+As              1     1.488   3.595  10.000   0.000  15.083    -0.083
+==========================================================================
+
+            Bond                   Population      Length (A)
+======================================================================
+         Ga 1 -- As 1                   0.34        2.44652
+======================================================================
+```
+
+Despite having the same structure, the results are very different now:
+
+- Unlike before, the Ga and As ions have charges on them - this is indicative of ionic/polar character
+- There is now a much smaller population in the Ga-As bond. This indicates less covalent character
+- Especially in the case of Ga, the s:p shell ratio is far off from 1:3 - this again indicates that the bonding is not like in the cases above
+
+## Diatomic molecules
+
+Next we will examine a few diatomic molecules - HF, HCl and HBr.
+
+Here is the ```HF.cell``` file
+
+```
+%block lattice_abc
+5 5 5
+90 90 90
+%endblock lattice_abc
+
+%block positions_abs
+H 2 2 2
+F 2.91 2 2
+%endblock positions_abs
+```
+
+Since we are just trying to look at a single diatomic molecule, the cell is defined rather simply - an arbitrarily large cube for the cell (making it too large would make the calculation take longer, but make it too small and it won't be accurate), 1 atom placed about in the middle, and the 2^nd^ atom placed a bond length away from it - in this case F is $0.91 \mathring{A}$ to the right of the H . The bond lengths can be found on a [database](https://cccbdb.nist.gov/), or you may perform a [geometry optimisation](../../..//documentation/Geometry_Optimisation/overview) to find it yourself if you wish.
```

Comment:
"Won't be accurate" is somewhat vague. You start to simulate a loose crystal rather than a disperse molecule.

Strictly don't need to write `2^nd^`, `2nd` will do the same and [mkdocs smartsymbols](https://facelessuser.github.io/pymdown-extensions/extensions/smartsymbols/) will handle that. 

`\mathring{A}` is a very ugly render for the Ångstrom (since the `A` is italicised in mathmode), prefer the unicode character or `\mathring{\text{A}}`. Unfortunately, we don't use the macros plugin which may be useful.

Double `/` in relative link may lead to issues with linking.